ecs-logs-collector.sh

```diff
@@ -574,31 +573,6 @@ get_ecs_agent_info() {
   fi
 }
 
-get_docker_containers_info() {
-  try "inspect all Docker containers"
-
-  mkdir -p "$info_system"/docker
-
-  if pgrep dockerd > /dev/null ; then
-    for i in $(docker ps -a -q); do
-      timeout 10 docker inspect "$i" > "$info_system"/docker/container-"$i".txt 2>&1
```

Comment:
How about we exclude every container but ECS Agent?

Reply:
This is a good idea, I'll work on this.

Reply:
Done